openprescribing/templates/spending_for_one_entity.html

```diff
@@ -33,6 +34,28 @@ <h1>{{ title }}</h1>
 
 <hr/>
 
+{% if form %}
+  {% if not signed_up_for_alert %}
+  <form method="post" class="form form-inline">
+   {% csrf_token %}
+   <p>
+     Get alerted by email when new price concession data is available.
```

Comment:
add something like "(up to 4 times per month)" (or whatever is accurate) so they know expected frequency?